src/components/Header/headerRightBlock/headerRightBlock.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { FaStar, FaSearch } from "react-icons/fa";
+import ModalWind from "../../modal-wind";
+import AuthComponent from "../../../security/authComponent";
+import DropdownMenu from "../../../containers/dropdownMenuContainer";
+import ToggleButton from "../../shared/toggleButton/toggleButton";
+import headerConstants from "../../../constants/headerConstants";
+import "./headerRightBlock.scss";
+
+const { SIGN_IN } = headerConstants;
+
+const HeaderRightBlock = user => {
+  const { id } = user.id ? user : {};
+
+  return (
+    <div className="header__right-block">
+      <div className="header-icons">
+        <div className="language-toggle">
```

Comment:
This div is redundant. You can pass `className` prop directly to the `ToggleButton` component

Reply:
yes, thak you